### PR TITLE
Add animation tutorial

### PR DESCRIPTION
**Pull request by violinyanev**

### CODE REVIEW

File: animations/manual.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+![](./docs/cube_anim.gif)
+
+This tutorial shows how to import animation data using the glTF format.
+
+## A general note on animation data
+
+Animation data can typically grow large and cause performance problems if not carefully optimized.
+glTF offers a good compromise between animation quality and exported data size. In order to utilize glTF
+to its best, it's adviced to get a good understanding how the different interpolation mechanisms works, and
```
Comment (bojackHaasman) [inline animations/manual.md:21]:
```suggestion
to its best, it's adviced to get a good understanding how the different interpolation mechanisms work, and
```

File: animations/manual.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+* In the animation panel, change the "End" frame from 250 to 60 to reduce the animation time
+* Select the 1st frame in the animation editor
+* Press 'i' for recording keyframes
+* From the dropdown menu, select "Rotate, scale, and translate" keyframe type
+* Enter some values in the property fields - e.g. set the scaling to 1, 2, 3
+* Move to frame 60
+* Press 'i' again, select some other values for the last keyframe (e.g. random rotation, translation and scale values)
+
+Next, we need to export the cube alongside its animation using the glTF export menu. For details how to export
+from Blender, check [the corresponding section basic cube tutorial](../hello_world/manual.md#Export-glTF-from-Blender).
```
Comment (bojackHaasman) [inline animations/manual.md:40]:
```suggestion
from Blender, check [the corresponding section of the basic cube tutorial](../hello_world/manual.md#Export-glTF-from-Blender).
```

File: animations/manual.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+If you uncollapse the "Animation" export menu, you will notice a setting called "Always sample animations". It's
+on by default, but what it causes is that every single frame results in a data point in the resulting glTF file.
+So if you have a simple linear animation over 120 frames, you will export 120 data points instead of just two (which would
+result in the exact same visual result as 120 key points). We suggest not using this feature of the exporter, unless really
+needed.
+
+## Importing the animation
+
+Let's import the animated cube in the Ramses Composer. If you haven't yet, check out the [section in the glTF import
+tutorual](../complex_import/manual.md#Scene-graph-and-resources) which explains how to import complex glTF files. When importing
```
Comment (bojackHaasman) [inline animations/manual.md:55]:
```suggestion
tutorial](../complex_import/manual.md#Scene-graph-and-resources) which explains how to import complex glTF files. When importing
```

**Review by bojackHaasman — CHANGES_REQUESTED** [review]

**Review by ParadoxDaKo — COMMENTED** [review]
OK. Concise information, good introduction and I was able to follow the tutorial. One spelling mistake noticed.

One thing though while we're at it: The Ramses object mapping table should probably be updated with the following values:

| Composer type     |From View          |Exported Ramses object(s)                              |Exported Logic object(s)   |Notes |
|-------------------|-------------------|-------------------------------------------------------|---------------------------|-------------------------------------|
|Animation |Scene              |                        | rlogic::AnimationNode | A direct mapping to an AnimationNode. The rlogic data for these AnimationNodes is held in Ramses Composer's AnimationChannels |
|AnimationChannel |Resources | | 2 or 4 rlogic::DataArrays, depending on interpolation type | AnimationChannels are clumps of rlogic::DataArrays that hold animation data for rlogic::AnimationChannel mapping |

**Comment by violinyanev** [comment]
> 

Nice catch! Added!

**Review by ParadoxDaKo — APPROVED** [review]

**Review by bojackHaasman — APPROVED** [review]

**Pull request merged**